src/payment/strategies/masterpass/masterpass-payment-strategy.spec.ts

```diff
@@ -0,0 +1,295 @@
+import { createClient as createPaymentClient } from '@bigcommerce/bigpay-client';
+import { Action } from '@bigcommerce/data-store';
+import createAction from '@bigcommerce/data-store/lib/create-action';
```

Comment:
`import { createAction, Action } from '@bigcommerce/data-store';` - you can combine with the line above.

Reply:
@michxff Let's address this comment it's a quick change.